src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,138 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private Node<T> head;
+    private Node<T> tail;
+    private int size;
+
+    public MyLinkedList() {
+        head = new Node<>(null, null, null);
+        tail = head;
+        size = 0;
+    }
+
+    private void increaseSize(int value) {
+        size += value;
+    }
```

Comment:
Private methods and classes should be after public ones in your class

see common mistakes https://mate-academy.github.io/jv-program-common-mistakes/java-core/collections/linked-list.html